.github/workflows/readme.yml

```diff
@@ -0,0 +1,20 @@
+name: Assemble project
+
+on:
+  push:
```

Comment:
The SDK is not under active development and it's open source. The build time is also very short. I don't think we can reach any limits with the `push` trigger and it allows us to work freely with feature branches.